oak_functions/README.md

```diff
@@ -54,20 +54,18 @@ from the conventional computing model.
 
 ### Oak Functions Loader and the Trusted Runtime
```

Comment:
Either "Oak Functions Launcher" or I think we can just remove the whole heading.